src/docfx/SubCommands/DocumentBuilderWrapper.cs

```diff
@@ -253,10 +259,7 @@ private static IEnumerable<DocumentBuildParameters> ConfigToParameter(BuildJsonC
                     GlobUtility.ExpandFileMapping(baseDirectory, pair.Value.GetFileMapping(FileMappingType.Overwrite)),
                     GlobUtility.ExpandFileMapping(baseDirectory, pair.Value.GetFileMapping(FileMappingType.Resource)));
                 parameters.VersionName = pair.Key;
-
-                // to-do: get changelist from config/compare file modify date
-                parameters.ChangeList = new ChangeList();
-
+                parameters.Changes = GetIntersectChanges(parameters.Files, changeList);
```

Comment:
got it~

---
In reply to: [75610363](https://github.com/dotnet/docfx/pull/625#discussion_r75610363) [](ancestors = 75610363,75596191)